hydrofunctions/hydrofunctions.py

```diff
@@ -122,14 +138,58 @@ def extract_nwis_df(response_obj):
         raise exceptions.HydroNoDataError("The NWIS reports that it does not"
                                           " have any data for this request.")
 
-    data = nwis_dict['value']['timeSeries'][0]['values'][0]['value']
-
+    # create lists of timeseries keys, names, and noDataValues
+    keys = []
+    names = []
+    noDataValues = []
+    for idx, tts in enumerate(ts):
+        keys.append(idx)
+        tag = tts['name'].split(':')[1]
+        tag += ' - '
```

Comment:
Perhaps the hyphen should get put into the `try` clause so that it only gets added to `tag` if we are able to extract the variable name from the response.

Reply:
According to one of our NWIS folks the site info for `dv` and `iv` data will always have the same keys except for `iv` data  which will not have a statistic key.